<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FunctionalRecoveryClass</class>
 <widget class="QMainWindow" name="FunctionalRecoveryClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FunctionalRecovery</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="MeshViewer" name="mesh_widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>581</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_load_mesh">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Mesh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_oversegement">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>460</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Oversegement</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>923</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MeshViewer</class>
   <extends>QWidget</extends>
   <header>meshviewer.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="functionalrecovery.qrc"/>
 </resources>
 <connections/>
</ui>
